client/extensions/woocommerce/app/settings/shipping/shipping-package-list.js

```diff
@@ -0,0 +1,61 @@
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import Button from 'components/button';
+import Card from 'components/card';
+import ShippingHeader from './shipping-header';
+import ShippingPackage from './shipping-package';
+
+class ShippingPackageList extends Component {
+	constructor( props ) {
+		super( props );
+
+		//TODO: use redux state and real data
+		this.state = {
+			packages: [ {
+				type: 'box',
+				name: 'Custom Box',
+				dimensions: '10 x 10 x 10 in'
+			}, {
+				type: 'envelope',
+				name: 'Custom Envelope',
+				dimensions: '12 x 9 x 0.5 in'
+			} ],
+		};
+	}
+
+	renderShippingPackage( pckg, index ) {
+		return ( <ShippingPackage key={ index } { ...pckg } /> );
+	}
+
+	render() {
+		const { translate } = this.props;
+
+		return (
+			<div>
+				<ShippingHeader
+					label={ translate( 'Packages' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **6** times:
`translate( 'Theme Packages' )` *ES Score: 9*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).